src/main/java/uk/ac/cam/alpha2017/airqualityradar/backend/data/entities/DataRowEntityColumns.java

```diff
@@ -0,0 +1,28 @@
+package uk.ac.cam.alpha2017.airqualityradar.backend.data.entities;
+
+public interface DataRowEntityColumns {
```

Comment:
This class is used effectively to store the string values of the column names.  If we made it into an enum, I think we'd need to add a method to give the string value for each column enum.  So I don't think it's possible to make it into an enum without making it more complicated.